src/SqlSquare/JoinType.purs

```diff
@@ -0,0 +1,19 @@
+module SqlSquare.JoinType where
+
+import Prelude
+
+data JoinType
+  = LeftJoin
+  | RightJoin
+  | InnerJoin
+  | FullJoin
+
+printJoinType ∷ JoinType → String
+printJoinType = case _ of
+  LeftJoin → "left join"
+  RightJoin → "right join"
+  FullJoin → "full join"
+  InnerJoin → "inner join"

```

Comment:
This is a general comment, but here seems like a good a place as any... should we render the SQL-syntax-stuff as all uppercase perhaps? That seems more conventional to me; and I tend to write it that way when doing so by hand as it makes it easier to spot the parts that are identifiers, etc.